distribution/docs/src/main/jdocs/content/_tables/org.codice.ddf.catalog.plugin.metacard.MetacardIngestNetworkPlugin-table-contents.adoc

```diff
@@ -0,0 +1,35 @@
+:type: documentation
+:status: published
+
+.[[org.codice.ddf.catalog.plugin.metacard.MetacardIngestNetworkPlugin]]Catalog Metacard Ingest Network Plugin
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Criteria
+|criteriaKey
+|String
+|Specifies the criteria for the test of equality; which value will be tested? IP Address? Hostname?
+|remoteAddr
+|true
+
+|Expected Value
+|expectedValue
+|String
+|The value that the criteria must equate to for the attribute overrides to occur.
+|
+|true
+
+|New Attributes
+|newAttributes
+|String"
+|New metacard attributes to apply; if an attribute is specified here, it will be placed on the metacard only if the attribute is not already set. The format should be 'key=value' where 'value' can be a comma-separated list of strings. Whitespace is ignored.
```

Comment:
If anyone is curious what the code looks like for this feature, you'll find the code handling the entire setter string and `=` splitting in the KeyValueParser:
https://github.com/codice/ddf/blob/master/catalog/plugin/catalog-plugin-metacardingest-network/src/main/java/org/codice/ddf/catalog/plugin/metacard/util/KeyValueParser.java#L91-L95

And the multi-valued attribute `,` splitting in AttributeFactory:
https://github.com/codice/ddf/blob/master/catalog/plugin/catalog-plugin-metacardingest-network/src/main/java/org/codice/ddf/catalog/plugin/metacard/util/AttributeFactory.java#L175